addons/acr/config.cpp

```diff
@@ -0,0 +1,163 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+            QGVAR(Rifleman),
+            QGVAR(Squad_Leader),
+            GVAR(Autorifleman),
+            QGVAR(Rifleman_LAT),
+            QGVAR(Sharpshooter),
+            QGVAR(Armour_Crewman),
+            QGVAR(Officer),
+            QGVAR(Signaller),
+            QGVAR(Heli_Pilot),
+            QGVAR(Sniper),
+            QGVAR(Crewman),
+            QGVAR(Jet_Pilot),
+            QGVAR(Rifleman_AA),
+            QGVAR(Paratrooper),
+            QGVAR(Machine_Gunner),
+            QGVAR(Rifleman_HAT),
+
+            QGVAR(Operator),
+
+            QGVAR(T72),
+            QGVAR(Pandur2),
+            QGVAR(BVP),
+            QGVAR(Buran_Transporter),
+            QGVAR(Buran_Medical),
+            QGVAR(Buran_Cargo),
+            QGVAR(Dingo_Mg),
+            QGVAR(Dingo_GL),
+            QGVAR(Tatra_Ammo),
+            QGVAR(Tatra_Transport),
+            QGVAR(Tatra_Fuel),
+            QGVAR(Tatra_MG),
+            QGVAR(Tatra_Repair),
+            QGVAR(RM_70),
+            QGVAR(Sholef),
+            QGVAR(Wiesel_AA),
+            QGVAR(Wiesel_Radar),
+
+            QGVAR(f35),
+            QGVAR(Gripen),
+            QGVAR(Mi35),
+            QGVAR(AH1Z),
+            QGVAR(Yabhon),
+            QGVAR(Li2),
+            QGVAR(Mi17),
+
+            QGVAR(Podnos),
+            QGVAR(ags),
+            QGVAR(M2_Raised),
+            QGVAR(M2_Low),
+            QGVAR(TOW)
+        };
+        weapons[] = {
+            QGVAR(CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D),
+            QGVAR(CUP_arifle_ACRC_EGLM_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D),
+            QGVAR(LMG_03_F_CUP_acc_FlashlightACE_optic_MRCO_2D),
+            QGVAR(CUP_srifle_SVD_wdl_CUP_optic_PSO_1_1_open),
+            QGVAR(CUP_arifle_ACRC_blk_556),
+            QGVAR(CUP_arifle_ACRC_blk_556_ACE_optic_MRCO_2D),
+            QGVAR(CUP_srifle_M2010_blk_ACE_optic_LRPS_2D),
+            QGVAR(UKSF_ACR_ACR_CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D_CUP_acc_FlashlightACE_optic_MRCO_2D),
+            QGVAR(CUP_lmg_MG3_rail_ACE_optic_MRCO_2D)
+        };
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "uksf_main",
+            "uksf_common",
+
+            // not sure if these are needed??
+            "ace_medical_treatment",
+            "A3_Weapons_F",
+            "CUP_Weapons_ACR",
+            "CUP_Creatures_Military_ACR",
+            "A3_Characters_F",
+            "CuratorOnly_Characters_F_OPFOR",
+            "CUP_Weapons_RPG7",
+            "CUP_Weapons_SVD",
+            "A3_Characters_F_Tank_Headgear",
+            "CUP_Weapons_CZ75",
+            "CUP_Weapons_M2010",
+            "CUP_Weapons_Stinger",
+            "CUP_Weapons_MG3"
```

Comment:
A3 base dependencies are added as part of the uksf main/common modules, so nothing from A3 is needed here.
For CUP, they have specific load order patch classes for ease of use. In this case it'd be:
`CUP_Weapons_LoadOrder`
`CUP_Creatures_People_LoadOrder`
and also any load order patches for the vehicles used (these can be found via a config dump)